File: stacker.java
import javax.swing.*;
class sta
{
public sta()
{
int stack[]=new int[5];
int i,top=-1;
}
public void push()
{
if(top==4)
JOptionPane.showMessageDialog(null,"OVERFLOW");
else
{
top++;
int m=Integer.parseInt(JOptionPane.showInputDialog("ENTER A NUMBER"));
stack[top]=m;
}
}
public void show()
{
for(i=0;i<=top;i++)
{
System.out.print(stack[i]+" ");
}
System.out.println();
}
public void pop()
{
if(top==-1)
JOptionPane.showMessageDialog(null,"EMPTY");
else
{
top--;
}
public static void main(String arg[])
{
int ch;
sta s1=new sta();
switch(ch)
{
case 1:s1.push();
case 2:s1.show();
case 3:s1.pop();
default:JOptionPane.showMessageDialog(null,"WRONG CHOICE");
}
}
}

File: stacker.py
stack=[]
def push(stack,val):
     if len(stack)<5:
          stack.append(val)
     else:
          print("Overflow")
i=1
def show(stack):
     print(stack)

def pop(stack):
     if len(stack)>0:
          stack.pop()
     else:
          print("Underflow")
while True:
     ch=int(input("1PUSH\n2SHOW\n3POP\n4QUIT\nEnter Choice :"))
     if(ch==1):
          n=int(input("Enter a number"))
          push(stack,n)

     elif(ch==2):
          show(stack)
     elif(ch==3):
          pop(stack)
     else:
          break

